Week1/solution1.py

```diff
@@ -0,0 +1,11 @@
+#Write a python script to enter any number and check its prime or not.
+num=int(input('Enter a number:'))
+if num > 1:
+    for i in range(2, int(num/2)+1):
+        if (num % i) == 0:
+            print(num, "is not a prime number")
+            break
+    else:
+        print(num, "is a prime number")
+else:
+    print(num, "is not a prime number")

```

Comment:
what should happen if number is 0 or negative?

Reply:
If a number is 0 or negative number, it will be not prime number.